lib/cocoapods/installer/xcode/pods_project_generator/app_host_installer.rb

```diff
@@ -83,6 +83,12 @@ def install!
 
           private
 
+          ALLOW_INSECURE_LOADS_FOR_TESTS = {
```

Comment:
Is this declared separate due to the fact that we want it to work on macOS as well?

Reply:
Yes, and tvOS

Reply:
maybe a more explicit variable name like `ADDITIONAL_INFO_PLIST_ENTRIES` (without the _IOS_)?

Reply:
Done.